README.md

```diff
@@ -521,6 +521,29 @@ Where the following variables need to be substituted:
 - `$URL_TO_GET_OIDC_TOKEN`: The URL of the local server endpoint to call to retrieve the OIDC token.
 - `$HEADER_KEY` and `$HEADER_VALUE`: The additional header key/value pairs to pass along the GET request to `$URL_TO_GET_OIDC_TOKEN`, e.g. `Metadata-Flavor=Google`.
 
+#### Using External Account Authorized User workforce credentials
+
+External account authorized user credentials allow you to sign in with a web browser to an external identity provider account via the
```

Comment:
Let's also link to the official docs: https://cloud.google.com/iam/docs/workforce-obtaining-short-lived-credentials#browser-based-sign-in